packages/quantic/force-app/solutionExamples/main/lwc/exampleInsightPanel/e2e/fixture.ts

```diff
@@ -24,6 +39,8 @@ export const testInsight = quanticBase.extend<InsightPanelE2EFixtures>({
 
     await insightSetup.waitForInsightInterfaceInitialization();
     await search.waitForSearchResponse();
+    // This is needed in order to ensure the console errors are properly captured.
+    await page.waitForTimeout(100);
```

Comment:
Actually what's probably happening is that fixtures are executed in parallel most likely since they're all async.

So by putting this `page.waitForTimeout` you're just making this fixture take "longer" than the console.error fixture.

It's not perfect, but the other options is actually to modify this search/InsightPanelObject fixture to add that console.error listener to it.